src/main/webapp/components/query-manager/add-query.tsx

```diff
@@ -0,0 +1,66 @@
+import Button from '@material-ui/core/Button'
+import Typography from '@material-ui/core/Typography'
+import SearchIcon from '@material-ui/icons/Search'
+import React from 'react'
+import { NavigationBarContext } from '../../nav-bar-context'
+import QueryEditorPopover from './query-editor-popover'
+import { AddQueryProps } from './types'
+const ReactDOM = require('react-dom')
+
+export default (props: AddQueryProps) => {
+  const [query, setQuery] = React.useState<{ id?: string }>({
+    id: new Date().toString(),
+  })
+  const { onSearch, QueryEditor } = props
+
+  const navBarLeftRef = React.useContext(NavigationBarContext)
```

Comment:
:pencil2: make this variable name consistent across other components. In some places it's `navLeftRef`